### ISSUE TITLE
Error with 1990 data

### ISSUE THREAD

**Issue by tim-ness** [Watts-College/cpp-528-fall-2021#22]
Hi Prof @castower 

As I said in another thread (Lab 03: Following Instructions... #17), I'm able to run part 2 of the tutorial just fine for 2000 and 2010. When I try switching to 1990 and 2000, I get about 2/3 of the way through in and then start hitting some errors, the first with pop00 (or pop00.x?) in the "Select Gentrification Variables" section. R Studio is telling me that pop00 doesn't exist, but I've already been using it earlier in the code. 
<img width="548" alt="Screenshot 2021-11-03 010500" src="https://user-images.githubusercontent.com/69982382/140026580-28c0bd4e-5bd5-4a7c-9628-e6cf4bf400f7.png">
<img width="521" alt="Screenshot 2021-11-03 010539" src="https://user-images.githubusercontent.com/69982382/140026588-b9c12ba4-ce4d-4184-8071-a470fd2968cd.png">

Then in the next section it tells me there is an error in the size of an object, but it's with Abeline TX and I have no idea what to do with that.

<img width="489" alt="Screenshot 2021-11-03 010607" src="https://user-images.githubusercontent.com/69982382/140026678-93f6eab0-6d23-4ed3-af34-787e0666869a.png">
<img width="466" alt="Screenshot 2021-11-03 010635" src="https://user-images.githubusercontent.com/69982382/140026685-a70e4d33-2579-437c-8435-409c2705d76b.png">

Am I supposed to be going through and changing everything from 2000 and 2010 to 1990 and 2000 in order to answer the questions? That's what I assumed but I'm not sure.

Again, same as my question in the "Lab 03: Following Instructions... #17" thread, I'm not sure if this should be in the RMD document for Lab 03 submission or in another file?

Thank you!

**Comment by AmyTrumpower** [comment]
Just a thought on the first question …. Should the pop00 that is directly below the pop90.x that you highlighted actually be “pop00.x”?

**Comment by castower** [comment]
Yes @tim-ness you will need to change the variables from 2010 to 1990.

As @AmyTrumpower noted, if you recall from our data dictionary in lab 02, the variable names can differ by year. 

You will need to take a look at your data set to ensure that the variable names you list for the 90s are correct.
